### DISCUSSION TITLE
Copilot used to code Odoo

### DISCUSSION THREAD

**Discussion by jcfernandez-890825** [odoo-ide/pycharm-odoo#354] [General]
I don't if this topic is something that's allowed here. But is something that I'm curious about.

I started using Tabnine and then switched to GitHub copilot when it was announced that it will be using GPT4, along with a lot of perks, though this extra features are only available for VS and yet to come to Pycharm 😭.

Anyway, I haven't been able to find a way to tag other people, but if possible I would like people who is using this amazing plugin to participate on this.

Regards,
JC

**Comment by trinhanhngoc** [comment]
Hi @jcfernandez-890825 I just turned your issue into a discussion. I think this is the better place for such topics.
